platform/admin/ui/src/main/webapp/templates/module/ModuleDetail.layout.handlebars

```diff
@@ -14,7 +14,7 @@
 <div class="page-layout full-height well">
     <div class="header clearfix">
         <div class="btn btn-success pull-right">
-            Export
+            Export System Configurations
```

Comment:
I'd vote for `Export Configuration` to make the button smaller, but let's see what @pklinef, @andrewkfiedler, @djblue, @figliold and @michaelmenousek have to say :-)